.github/workflows/codeql.yml

```
@@ -0,0 +1,15 @@
+on:
+  push: { branches: [main] }
+  pull_request:
+  schedule: [{ cron: '0 6 * * 1' }]
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    permissions: { security-events: write }
+    strategy: { matrix: { language: [ 'python', 'javascript' ] } }
+    steps:
+      - uses: actions/checkout@v4
+      - uses: github/codeql-action/init@v3
```

actionlint (1.7.8)

could not parse as YAML: did not find expected ',' or '}'

(syntax-check)